istioctl/cmd/proxystatus.go

```diff
@@ -71,8 +90,41 @@ Retrieves last sent and last acknowledged xDS sync from Pilot to each Envoy in t
 			return sw.PrintAll(statuses)
 		},
 	}
-)
+
+	statusCmd.Flags().BoolVarP(&sdsDump, "sds", "s", false,
+		"retrieve synchronization between active secrets on Envoy instance with those on corresponding node agents")
+	statusCmd.Flags().BoolVar(&sdsJSON, "json", false,
```

Comment:
How does this interact with the original command?

Reply:
At the moment it's ignored from the main command. Another option is to have `--sds` and `--sds-json`, but I'm not sure we can do much better without making a subcommand.

Reply:
I think I prefer the `--sds` and `--sds-json` as it's more specific. It won't result in unexpected behaviour for other commands.

Reply:
fixed 👍